microsoft-edge/webview2/samples/webview2browser.md

```diff
@@ -0,0 +1,873 @@
+---
+title: WebView2Browser (Win32 C++/JS)
+description: "A web browser built with the Microsoft Edge WebView2 control."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 07/18/2022
+---
+# WebView2Browser (Win32 C++/JS)
```

Comment:
Title should be Win32 sample WebView2Browser

Reply:
changed globally